src/frequenz/sdk/timeseries/logical_meter/_resampled_formula_builder.py

```diff
@@ -28,6 +29,7 @@ def __init__(
         Args:
             namespace: The unique namespace to allow reuse of streams in the data
                 pipeline.
+            formula_name: A name for the formula.
             channel_registry: The channel registry instance shared with the resampling
```

Comment:
Could you please describe what is the difference between namespace and formula_name here?

Reply:
Namespace is used for grouping in the data pipeline classes - resampling and data sourcing actors.  The formula_name is a name for the formula the formula builder is going to build.